Sources/StytchCore/StytchClient/StytchClient.swift

```diff
@@ -38,9 +38,10 @@ public struct StytchClient: StytchClientType {
      - Parameters:
        - publicToken: Available via the Stytch dashboard in the `API keys` section
        - hostUrl: Generally this is your backend's base url, where your apple-app-site-association file is hosted. This is an https url which will be used as the domain for setting session-token cookies to be sent to your servers on subsequent requests. If not passed here, no cookies will be set on your behalf.
+       - dfppaDomain: The domain that should be used for DFPPA
      */
     public static func configure(publicToken: String, hostUrl: URL? = nil, dfppaDomain _: String? = nil) {
```

Comment:
is the `_` in `dfppaDomain _` a problem? I see that that's different from B2B and forget what that means in Swift!

Reply:
Yes that's wrong, fixed.